composer.json

```diff
@@ -13,16 +13,19 @@
 		"automattic/jetpack-autoloader": "^2.9.1"
 	},
 	"require-dev": {
-		"phpunit/phpunit": "7.5.20",
 		"woocommerce/woocommerce-sniffs": "0.1.0",
-		"suin/phpcs-psr4-sniff": "^2.2"
+		"suin/phpcs-psr4-sniff": "^2.2",
+		"bamarni/composer-bin-plugin": "^1.4"
```

Comment:
Removed phpunit, the `composer-bin-plugin` handles this now by using the composer file under `bin/composer/phpunit`, it will use a custom one for running tests in PHP8